### PR TITLE
Add Compose projects, README, and scripts

### PR DESCRIPTION
**Pull request by alanking**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,49 @@
...
+It is *highly recommended* to use a `virtualenv` python virtual environment. You can set one up which installs the Minimum Requirements (see above) like this:
+```bash
+virtualenv -p python3 ~/irods_testing_environment
+source ~/irods_testing_environment/bin/activate
+pip install docker-compose
+pip freeze
+```
+Compare the output to `requirements.txt`.
+
+To stand up the latest released versoin of iRODS in a Zone running on Ubuntu 18.04 using a Postgres 10.12 database to host the catalog, run the following:
```
Comment (trel) [inline README.md:27]: version

**Review by trel — COMMENTED** [review]
Don’t worry about linking to an issue.  Just update the commit message to be the way you want it.

File: README.md (hunk 1 above)
Comment (alanking) [reply README.md]: Done

**Comment by alanking** [comment]
Done. Also got the `/etc/irods/hosts_config.json` stuff worked out for 4-2-stable. Doesn't quite work for testing yet but at least redirection with different hostnames is possible

**Pull request merged**